deployment-scripts/ghidra_deploy/ghidra_deploy.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Set non-interactive installation mode
+export DEBIAN_FRONTEND=noninteractive
+
+echo "ghidrauser:ghidrauser" | sudo chpasswd
+export SUDO_PASS="ghidrauser"
```

Comment:
Problem: The password is hardcoded in the script, which poses a security risk.
Suggestion: Use a variable to store the password and ensure the script is stored in a secure location. Consider reading the password from a secure location, such as an environment variable.